plugin/Gui/AdminPageApiSettings.php

```diff
@@ -175,31 +176,46 @@ private function addFormModelGoogleMapsKey()
-				 </p>
-				 <details>
-					<summary>'.esc_html__('Available custom fields','onoffice-for-wp-websites').'</summary>
-					   <p>'.esc_html__('When this option is active, the plugin makes the following custom fields available in the detail view.'
-				.' These custom fields can be used in SEO plugins to fill out the title and description with the information of the currently shown estate.'
-				." For information on how to use custom fields consult you SEO plugin's documentation.",'onoffice-for-wp-websites').'<br>'.
-			esc_html__('These custom fields are only available in the detail view and on no other page.','onoffice-for-wp-websites').'<br>
-					   - '. esc_html__('Title (onoffice_title)','onoffice-for-wp-websites').'<br>
-					   - '. esc_html__('Description (onoffice_description)','onoffice-for-wp-websites').'<br>
-					   - '. esc_html__('Place (onoffice_place)','onoffice-for-wp-websites').'<br>
-					   - '. esc_html__('Postal code (onoffice_postal_code)','onoffice-for-wp-websites').'<br>
-					   - '. esc_html__('Property class (onoffice_property_class)','onoffice-for-wp-websites').'<br>
-					   - '. esc_html__('Marketing method (onoffice_marketing_method)','onoffice-for-wp-websites').'<br>
-					   - '. esc_html__('Data Record Ref No. (onoffice_id)','onoffice-for-wp-websites').'<br>
-				 </details>
-			</div>';
-		$descriptionNoticeSeo = '
-				 <p id="notice-seo">
-					'. esc_html__('We have detected an active SEO plugin: Yoast SEO. This option can lead to conflicts with the SEO plugin.','onoffice-for-wp-websites').'<br>
-					'. esc_html__('We recommend that you configure the onOffice plugin to not modify the title and description.','onoffice-for-wp-websites').'
-				 </p>';
+		$pluginSEO = [];
+		if (count(array_intersect(["wpseo/wp-seo.php"], get_option("active_plugins"))) > 0){
+			$pluginSEO['wp_seo'] = 'wpSEO';
+		}
+		if (count(array_intersect(["seo-by-rank-math/rank-math.php"], get_option("active_plugins"))) > 0){
+			$pluginSEO['rank_math_seo'] = 'Rank Math SEO';
+		}
+		if (count(array_intersect(["wordpress-seo/wp-seo.php"], get_option("active_plugins"))) > 0){
+			$pluginSEO['yoast_seo'] = 'Yoast SEO';
+		}
+		$listNamePluginSEO = implode(", ",$pluginSEO);
+		$titleDoNotModify = esc_html__("This plugin will not modify the title and description. This enables other plugins to manage those tags.",'onoffice-for-wp-websites');
+		$summaryDetailDoNotModify =  esc_html__('Available custom fields','onoffice-for-wp-websites');
+		$descriptionDetailDoNotModify = esc_html__('When this option is active, the plugin makes the following custom fields available in the detail view. These custom fields can be used in SEO plugins to fill out the title and description with the information of the currently shown estate. For information on how to use custom fields consult you SEO plugin\'s documentation.
+							These custom fields are only available in the detail view and on no other page.
+								\- Title (onoffice_title)
```

Comment:
I would like the list to appear as an actual HTML list. So the proper Markdown syntax would be straight `-` without the escape `\-`.

![grafik](https://user-images.githubusercontent.com/90763364/169303968-50715759-52ca-46d9-ab1e-c00b361fa661.png)

I tried this and noticed that there are some CSS rules that remove the list styling. The Firefox inspector says the rules are coming from https://testinstanz-jmaa.onofficeweb.com/wp-admin/load-styles.php?c=1&dir=ltr&load%5Bchunk_0%5D=dashicons,admin-bar,common,forms,admin-menu,dashboard,list-tables,edit,revisions,media,themes,about,nav-menus,wp-pointer,widgets&load%5Bchunk_1%5D=,site-icon,l10n,buttons,wp-auth-check,wp-jquery-ui-dialog&ver=5.9.3. The result is this:

![grafik](https://user-images.githubusercontent.com/90763364/169304044-be2d4619-fcbb-41b9-900b-6a44419e9f5c.png)

Could you check if you can make the list show up as in the first image? But don't try to fight the existing CSS (avoid `!important`). If it takes more than an hour, tell me and we'll find a different solution.